OpenRiaServices.DomainServices.Client/Test/Desktop/Data/QueryTests.cs

```diff
@@ -449,24 +449,27 @@ public void LoadOperation_MulipleCompletionBehavior()
             Cities.CityDomainContext cities = new CityDomainContext(TestURIs.Cities);
 
             var query = cities.GetCitiesQuery().Take(3);
-            LoadOperation lo = cities.Load(query, false);
+            LoadOperation<City> lo = cities.Load(query, false);
 
             lo.Cancel();
-
-            ExceptionHelper.ExpectInvalidOperationException(delegate
+            EnqueueConditional(() => lo.IsComplete);
+            EnqueueCallback(delegate
             {
-                lo.Cancel();
-            }, Resources.AsyncOperation_AlreadyCompleted);
+                ExceptionHelper.ExpectInvalidOperationException(delegate
+                {
+                    lo.Cancel();
+                }, Resources.AsyncOperation_AlreadyCompleted);
```

Comment:
Consider adding new error message if we should not be able to cancel while cancellation is pending

Reply:
will allow multiple calls to Cancel while Cancellation is pending